src/components/TransactionTable/TransactionTable.types.ts

```diff
@@ -0,0 +1,12 @@
+import { TransactionInterface } from '@models/transaction';
+
+export interface TransactionTablePropsInterface {
+    data: TransactionInterface[];
+}
+
+export interface StatusInfo {
+    label: string;
+    variant: 'success' | 'error' | 'info';
```

Comment:
if we are not writing any custom styles related to variants, we shall no restrict the variants.
```suggestion
    variant: BadgeProps['color'];
```
import BadgeProps from mui